examples/usecsharpdll/usecsharpdll.py

```diff
@@ -2,8 +2,10 @@
 example on how to load an external c# dll and use its functions.
 calculates distance between two 1D points
 """
+import Grasshopper
+path = Grasshopper.Folders.AppDataFolder
 import clr
-clr.AddReferenceToFileAndPath(r"C:\Users\chwaibel\Documents\GitHub\honey-badger\examples\usecsharpdll\SolarModel.dll") # path of your dll
+clr.AddReferenceToFileAndPath(path + "Libraries\SolarModel.dll") # need to put that dll into your Grasshopper libraries folder
```

Comment:
@christophwaibel the pythonic way to write this is `clr.AddReferenceToFileAndPath(os.path.join(path, "Libraries", "SolarModel.dll")`

Reply:
sure!